src/spell_parser.cpp

```diff
@@ -1584,7 +1584,22 @@ ACMD(do_cast) {
 
         /* Show chant messages. */
         start_chant(ch);
-        WAIT_STATE(ch, ch->casting.casting_time * PULSE_VIOLENCE / 2);
+
+        /* Set spellcasting delay */
+        if (ch->casting.casting_time > 3)
+            WAIT_STATE(ch, ch->casting.casting_time * PULSE_VIOLENCE / 2);
```

Comment:
This code is going away with the new code.

Reply:
OH.  This is the whole point of the revision.  This is where the casting "queue" is falling apart.